internal/osbuild/qemu_assembler.go

```diff
@@ -22,7 +22,7 @@ type QEMUPartition struct {
 	Size       uint64         `json:"size,omitempty"`
 	Type       *uuid.UUID     `json:"type,omitempty"`
 	Bootable   bool           `json:"bootable,omitempty"`
-	UUID       *uuid.UUID     `json:"uuid,omitempty"`
```

Comment:
Hm, the `Type` two lines above uses `uuid.UUID`, I think the type-safety is nicer to avoid typos. Combined with `MustParse()`, that is. Not a blocker though.

Reply:
Maybe this can be a follow up and also change `PTUUID`?

Reply:
I agree :)